Update readmes (#59)
Sindri-Labs/sindri-resources: circuit_tutorials/halo2/axiom-v0.3.0/README.md

# Halo2 Axiom v0.3.0 Circuit Tutorials

> In this directory, we provide a handful of Halo2 Axiom v0.3.0 circuits tutorials which have been prepared to be used with Sindri.

Reviewer: ```suggestion
In this directory, we provide a handful of Halo2 Axiom v0.3.0 circuit tutorials which have been prepared for use with Sindri.
```
